src/app/components/FAQ/FAQ.tsx

```diff
@@ -12,8 +12,8 @@ export const FAQ: React.FC<FAQProps> = () => {
 
   return (
     <div className="container mx-auto flex flex-col gap-2 p-6">
-      <h3 className="mb-4 font-bold">FAQ</h3>
-      <div className="flex flex-col gap-4">
+      <h3 className="mb-8 font-normal text-4xl text-primary">FAQ’s</h3>
```

Comment:
You can use `core-ui` here
```suggestion
      <Heading as="h3" variant="h4" className="mb-8">FAQ’s</Heading>
```